README.md

```diff
@@ -41,11 +42,11 @@ To create a migration, execute `db-migrate create` with a title. `node-db-migrat
 
 ```javascript
 exports.up = function (db, callback) {
-  callback();
```

Comment:
Well actually, you're breaking the migrations, as if there is no callback it will hang up and never end.

The templates are just a boilerplate.